source/switching_to_live/before_you_switch_to_live/index.html.md.erb

```diff
@@ -7,80 +7,89 @@ weight: 99
 
 Before you start taking live payments, you should:
 
-* have a signed MOU or contract with GOV.UK Pay
+* have a signed memorandum of understanding (MOU) or contract with GOV.UK Pay
```

Comment:
I think it needs to be clearer that this happens as part of the go-live flow. They can download the contract/ MoU from a test account for review, but there's no need to sign it before starting the go-live journey